rgw/v2/utils/utils.py

```diff
@@ -170,7 +194,7 @@ def get_all_in_dir(path):
 
 def gen_bucket_name_from_userid(user_id, rand_no=0):
     log.info('generating bucket name or basedir to create')
-    bucket_name_to_create = user_id + "_" + BUCKET_NAME_PREFIX + "_" + str(rand_no)
+    bucket_name_to_create = user_id + "-" + BUCKET_NAME_PREFIX + "-" + str(rand_no)
```

Comment:
using `_`  worked in nautilus. If this is not working in rhcs 5.0. we need to fie a BZ. can you add the bz here as a comment once you file it.

Reply:
will wait for the bz before we merge.

Reply:
filed BZ  https://bugzilla.redhat.com/show_bug.cgi?id=1942136, to track the issue